src/components/Chart.js

```diff
@@ -1,44 +1,85 @@
-import React, {Component} from "react";
-import ChartComponent, {Line} from "react-chartjs-2";
-
+import React, { Component } from "react";
+import ChartComponent, { Line } from "react-chartjs-2";
+import moment from "moment";
 
 class Chart extends Component {
-    constructor(props) {
-        super(props);
-        let values = props.values;
+  constructor(props) {
+    super(props);
+    const currentMonthDates = Array.from(
+      { length: moment().daysInMonth() },
+      (x, i) => moment().startOf("month").add(i, "days")
+    );
+    let data = () => props.values.map((arr) => ({ t: arr[0], y: arr[1] }));
```

Comment:
You can rewrite it to:
let data = props.values.map(([time, value]) => ({t: time, y: value}))

and then you don't need to use data(), just data.